gateway-service-knoxsso/src/main/java/org/apache/knox/gateway/service/knoxsso/WebSSOResource.java

```diff
@@ -95,7 +95,7 @@ public class WebSSOResource {
   private static final String ORIGINAL_URL_COOKIE_NAME = "original-url";
   private static final String DEFAULT_SSO_COOKIE_NAME = "hadoop-jwt";
   private static final String SSO_COOKIE_SAMESITE_DEFAULT = "Strict";
-  private static final long TOKEN_TTL_DEFAULT = 30000L;
+  private static final long TOKEN_TTL_DEFAULT = 15000 * 60;
```

Comment:
@pzampino Right, this might not be strictly needed. We can modify the deployment related default configs only. 
But I think it shouldn't break anything either. If someone needs a specific value I would expect them to have an explicit configuration instead of relaying the default. And the default 30seconds timeout seems to be too low anyway, isn't it?

Reply:
This does seem to be a more reasonable default, so let's make this change.